src/commands/codepush/deployment/ls.ts

```diff
@@ -0,0 +1,30 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp } from "../../../util/commandline";
+import { out } from "../../../util/interaction";
+import { DefaultApp } from "../../../util/profile";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest } from "../../../util/apis";
+const _ = require("lodash");
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:deployments:ls");
+
+@help("List the deployments associated with an app")
+export default class LsCommand extends AppCommand {
```

Comment:
Why are there two files here at all? They call the same back end and display the results in the same format. If there's a difference between the two that I missed it's probably better to vary with an option rather than a completely separate command.

If it's just to get both `list` and `ls` to do the same thing, don't do that. We use `list` everywhere in the CLI, we don't use `ls` for anything.